README.md

```diff
@@ -270,6 +281,9 @@ EOF
 $ aws sqs create-queue --queue-name "${SQS_QUEUE_NAME}" --attributes file:///tmp/queue-attributes.json
 ```
 
+If you are sending Lifecycle termination events from ASG directly to SQS, instead of through EventBridge, then you will also need to create an IAM service role to give Amazon EC2 Auto Scaling access to your SQS queue. Please follow [these linked instructions to create the IAM service role: link.](https://docs.aws.amazon.com/autoscaling/ec2/userguide/configuring-lifecycle-hook-notifications.html#sqs-notifications)
```

Comment:
We should aim to be less verbose and with fewer redirects wrt documentation and setup. Is it possible to re-work these setup steps so users don't need to bounce between them? SQS + IAM setup stand out to me as constants so maybe start with those and deviate from there (whether or not users want EventBridge)? OR start with EventBridge and let steps flow from there.

Applies to above entry as well